XenAdmin/PasswordsRequest.cs

```diff
@@ -113,10 +113,6 @@ internal static void HandleRequest(string destdir)
                     log.WarnFormat("Valid token, but for the wrong app. Will re-prompt");

```

Comment:
Nit: This instance of `WarnFormat` should be `Warn` as it doesn't have any other parameters. It was already like that though, nothing you've changed.

Reply:
A quick grep shows a few more instances in other files; also a few instances of the other related methods like DebugFormat etc. I'll raise a PR at some point to fix all of them.